src/components/multi-action-button/multi-action-button.style.ts

```diff
@@ -77,12 +77,36 @@ const StyledButtonChildrenContainer = styled.div<StyledButtonChildrenContainerPr
     white-space: nowrap;
     z-index: ${theme.zIndex.popover};
     box-shadow: var(--boxShadow100);
+    border-radius: var(--borderRadius100);
 
     ${StyledIcon} {
       margin-left: 0;
       left: 8px;
     }
 
+    > {
```

Comment:
This styling will break if the middle or last child button has an `href` set 

![image](https://user-images.githubusercontent.com/44157880/228317923-ee77dea1-48c0-445f-9a9c-2e5e2257c0a7.png)

Something like the below will work: 

```
 > {
            &:first-child:not(:last-child) {
              border-top-left-radius: var(--borderRadius100);
              border-top-right-radius: var(--borderRadius100);
              border-bottom-right-radius: var(--borderRadius000);
              border-bottom-left-radius: var(--borderRadius000);
            }
            &:not(:first-child):not(:first-child) {
              border-radius: var(--borderRadius000);
            }
            &:last-child:not(:first-child) {
              border-top-right-radius: var(--borderRadius000);
              border-top-left-radius: var(--borderRadius000);
              border-bottom-left-radius: var(--borderRadius100);
              border-bottom-right-radius: var(--borderRadius100);
            }
          }
          ```